lib/src/darwin.dart

```diff
@@ -404,20 +431,21 @@ class AVAudioSessionInterruptionOptions {
   int get hashCode => value.hashCode;
 }
 
-///// The route change reasons for [AVAudioSession].
-//enum AVAudioSessionRouteChangeReason {
-//  unknown,
-//  newDeviceAvailable,
-//  oldDeviceUnavailable,
-//  categoryChange,
-//  override,
-//  wakeFromSleep,
-//  noSuitableRouteForCategory,
-//  routeConfigurationChange,
-//}
-//
-//enum AVAudioSessionSilenceSecondaryAudioHintType { begin, end }
-//
+/// The route change reasons for [AVAudioSession].
+enum AVAudioSessionRouteChangeReason {
+  unknown,
+  newDeviceAvailable,
+  oldDeviceUnavailable,
+  categoryChange,
+  override,
+  wakeFromSleep,
+  noSuitableRouteForCategory,
+  routeConfigurationChange,
+}
+
+/// The interruption types for [AVAudioSessionSilenceSecondaryAudioHint].
+enum AVAudioSessionSilenceSecondaryAudioHintType { began, end }
+
 //class AVAudioSessionRouteDescription {
 //  final List<AVAudioSessionPortDescription> inputs;
 //  final List<AVAudioSessionPortDescription> outputs;
```

Comment:
We just need "outputs" for now.

Reply:
And then in `AVAudioSessionPortDescription` we only need `portType`.

Reply:
Had some issues here because of multiple possible routes here. Ended up checking the port type in DarwinAudioSession.m and passing `shouldInterrupt`